common/djangoapps/entitlements/tests/test_models.py

```diff
@@ -0,0 +1,92 @@
+"""Test Entitlements models"""
+
+import unittest
+from datetime import datetime, timedelta
+
+import pytz
+from django.conf import settings
+from django.test import TestCase
+
+from openedx.core.djangoapps.content.course_overviews.tests.factories import CourseOverviewFactory
+from student.tests.factories import CourseEnrollmentFactory
+
+# Entitlements is not in CMS' INSTALLED_APPS so these imports will error during test collection
+if settings.ROOT_URLCONF == 'lms.urls':
+    from entitlements.tests.factories import CourseEntitlementFactory
```

Comment:
What will happen when the test code references `CourseEntitlementFactory` in a non-lms environment?

Reply:
Do we run common tests in both environments? I know we run some openedx tests like that.

Reply:
NVM, I see you can specify tests that should only be run in the LMS environment. I didn't know that was possible. Good to know going forward.

Reply:
It fails. There's precedent in the `test_views.py` as this code isn't supposed to run outside of LMS.